src/main/java/frc/robot/subsystems/transfer/TransferSubsystem.java

```diff
@@ -22,12 +22,15 @@ public enum TransferPreset {
 
   private CANSparkMax transferMotor;
 
+  private IdleMode mode;
+
   @Override
   public void initialize() {
     if (ENABLED) {
       transferMotor = new CANSparkMax(RobotConstants.PORTS.CAN.TRANSFER, MotorType.kBrushless);
       transferMotor.setInverted(false);
       transferMotor.setIdleMode(IdleMode.kCoast);
+      mode = IdleMode.kCoast;
```

Comment:
Should the transfer subsystem have its motors in brake mode?  I can see arguments either way.  The main one being if the transfer can keep up with the shooter or not.  If it's in brake mode, it will impart more drag on the note shot; but it might do a better job of holding the node in place and give fewer mis-feeds.